NavigationReactNative/src/ios/NavigationModule.m

```diff
@@ -35,13 +35,19 @@ - (dispatch_queue_t)methodQueue
         [navigationController popToViewController:navigationController.viewControllers[crumb] animated:true];
     }
     if (crumb > currentCrumb) {
-        NSMutableArray *controllers = [navigationController.viewControllers mutableCopy];
+        NSMutableArray *newControllers = [[NSMutableArray alloc] init];
         for(NSInteger i = 0; i < crumb - currentCrumb; i++) {
             NSInteger nextCrumb = currentCrumb + i + 1;
             UIViewController *controller = [[NVSceneController alloc] initWithModule: self crumb: nextCrumb tab: tab title: titles[nextCrumb] appKey: appKey];
-            [controllers addObject:controller];
+            [newControllers addObject:controller];
+        }
+        
+        if (newControllers.count == 1) {
```

Comment:
Check if `crumb - currentCrumb == 1` instead of `newControllers.count == 1` so that it matches the [similar check in the Android `NavigationModule`](https://github.com/grahammendick/navigation/blob/6e18b43e41479f7a795ef1ce29a97483e36ade63/NavigationReactNative/src/android/app/src/main/java/com/navigation/reactnative/NavigationModule.java#L115)